v/view_password/view_password_6.0.0_ubi8.5.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# -----------------------------------------------------------------------------
```

Comment:
Please add the `-ex` option here itself if possible. Please refer this [template](https://github.com/ppc64le/build-scripts/blob/41f2b7a8fbefec28c76905652f5e7c8d5b4484b8/templates/build_script_php.sh#L1). Also, try to avoid modifications to the metadata section - extra line at line 2. The Travis check might recognize similar modifications and the Travis check might fail.